test/Microsoft.TestPlatform.AcceptanceTests/TranslationLayerTests/EventHandler/DiscoveryEventHandler.cs

```diff
@@ -71,9 +71,11 @@ public class DiscoveryEventHandler2 : ITestDiscoveryEventsHandler2
     /// </summary>
     public List<TestCase> DiscoveredTestCases { get; }
 
-    public IList<DiscoveredSource> FullyDiscoveredSources { get; private set; }
-    public IList<DiscoveredSource> PartiallyDiscoveredSources { get; private set; }
-    public IList<DiscoveredSource> NotDiscoveredSources { get; private set; }
+    public IList<string>? FullyDiscoveredSources { get; private set; }
```

Comment:
Is it some merge issue or do you mean to have both sets of properties? (string + DiscoveredSource).